spot_wrapper/wrapper.py

```diff
@@ -5,14 +5,10 @@
 import time
 import traceback
 import typing
-from enum import Enum
-from collections import namedtuple
-from dataclasses import dataclass, field
```

Comment:
Apologies for the late response. You still need to import the dataclass because https://github.com/heuristicus/spot_wrapper/blob/be0e81e551f29f9ba7a43642e304831374478f9a/spot_wrapper/wrapper.py#L426 references it. Other than that, LGTM.